### PR TITLE
Feature/1750 zaaktype procestype

### LINKED ISSUE #1750: When creating test data with generate_data, zaaktypen have empty Selectielijst procestype
### Product versie / Product version

Version 1.13.0-archief.0 

### Omschrijf het probleem / Describe the bug

All zaaktypen created with the management command `generate_data` have `selectielijst_procestype = ""`.

### Stappen om te reproduceren / Steps to reproduce

_No response_

### Verwacht gedrag / Expected behavior

For Open archiefbeheer it would be great if this was a URL pointing to a procestype in the selectielijst API

### PR DESCRIPTION
**Pull request by annashamray**
Fixes #1750

### CODE REVIEW

File: src/openzaak/management/commands/generate_data.py [hunk 1]
```diff
@@ -250,6 +265,7 @@ def generate_catalogi(self):
         # zaaktype - 100
         zaaktypen = ZaakTypeFactory.build_batch(
             self.zaaktypen_amount,
+            selectielijst_procestype=random.choice(list(self.sl_result_mapping.keys())),
```
Comment (SilviaAmAm) [inline src/openzaak/management/commands/generate_data.py:268]: Maybe I understand wrong, but I thought that the procestype referenced by the zaaktype should be from this endpoint: https://selectielijst.openzaak.nl/api/v1/procestypen and not from the resultaten endpoint?

Comment (annashamray) [reply src/openzaak/management/commands/generate_data.py:268]: `Resultaat` resource in Selectielijst API has a FK to `Procestype` 
So here we request `resultaten` endpoint, retrieve `procestype` and `url` attributes and create a dictionary with them. 

The idea is to add some random `procestypes` to zaaktypen and then to add related `resultaten` to the zaken

Comment (SilviaAmAm) [reply src/openzaak/management/commands/generate_data.py:268]: Ahh I see!

**Review by SonnyBA — APPROVED** [review]
:+1:

**Pull request merged**